driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -64,7 +64,33 @@ public static Bson addFields(final Field<?>... fields) {
      * @since 3.4
      */
     public static Bson addFields(final List<Field<?>> fields) {
-        return new AddFieldsStage(fields);
+        return new FieldsStage("$addFields", fields);
+    }
+
+    /**
+     * Creates an $accumulator pipeline stage
+     *
+     * @param fields        the fields to add
+     * @return the $accumulator pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/accumulator/ $accumulator
+     * @mongodb.server.release 4.4
+     * @since 4.4
+     */
+    public static Bson accumulator(final Field<?>... fields) {
```

Comment:
As the accumulator operator can only take certain field names, they should be explicitly listed. Otherwise users could easily create an invalid accumulator expression.